api/client/options.proto

```diff
@@ -62,6 +62,15 @@ message SequencerIdleStrategy {
   SequencerIdleStrategyOptions value = 1;
 }
 
+message H1ConnectionReuseStrategy {
+  enum H1ConnectionReuseStrategyOptions {
+    DEFAULT = 0;
```

Comment:
Best to make that `NOT_SET` and reject IMHO, to avoid surprising behavior if defaults change.

Reply:
Rephrasing to make sure I understand, are you proposing we make this a mandatory option?

Reply:
Yep, that way you don't get any surprises when defaults change in different endpoints.

Reply:
Ping on this one.

Reply:
See https://github.com/envoyproxy/nighthawk/pull/198#issuecomment-569518828; I was thinking that maybe it is better to we postpone this until we land the non-experimental (Envoy-based) approach?